gen/sam/sam_sa10.py

```diff
@@ -54,7 +54,7 @@ def generate(self):
         )
```

Comment:
Recommend reducing SRs and TRs to 1 each. Not sure why these generators have 2, tbh.

Reply:
There are two types of search radar for the grumble. Aiui one of them is specifically for low altitude detection, and the other is for the rest, so you need both for coverage.

Agreed for the track radar though.